CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 To Be Released
 ===============
 
+* Updated photos plugin integration to show lightbox
```

Comment:
This line can be removed. B/c we haven't done a release of photos plugin support. So our next release is going to have photos+lightbox

Reply:
I've updated the line to contain both

Reply:
About being optional, we can define a new variable that loads or not the new sections added in the base.html would that be ok?

Reply:
See my new review https://github.com/Pelican-Elegant/elegant/pull/351#discussion_r292445642